### DISCUSSION TITLE
Glosario: Definiciones

### DISCUSSION THREAD

**Discussion by aloyber** [gcba/estandares#149] [Ideas] [answered]
La idea del glosario surge para unificar las definiciones que tenemos los integrantes del equipo de Experiencia Digital y utilizar los mismos términos. Además podemos empezar a investigar y analizar la percepción que tienen los ciudadanos sobre estos términos para categorizar y organizar la información de una forma intuitiva.

Podemos empezar nombrando las palabras que tenemos para luego hacer un análisis mas profundo sobre cada una de ellas y empezar a categorizarlas, entre ellas: transacción, trámite, solicitud, turno, pago, inscripción, registro, consulta, servicio, calendario, evento, taller, actividad y programa.

Algunas definiciones de **trámite**:
- Paso que, junto con otros, debe realizarse de forma sucesiva para solucionar un asunto que requiere un proceso.
_"los trámites del divorcio"_
- Un trámite se define como cualquier solicitud o entrega de información que las personas físicas o morales del sector privado hacen ante una dependencia u organismo descentralizado, ya sea para cumplir una obligación, obtener un beneficio o servicio o, en general, a fin de que se emita una resolución, así como cualquier documento que dichas personas estén obligadas a conservar. _Fuente: Gobierno de México_
- Un trámite es una actuación que implica una acción o conjunto de acciones para lograr obtener un beneficio o cumplir con una obligación. Se realizan trámites de todo tipo en ámbitos de la educación, legales, financieros, o de salud entre otros.

Algunas definiciones de **transacción**:
- Trato o convenio por el cual dos partes llegan a un acuerdo comercial, generalmente de compraventa. _"transacciones comerciales"_
- La transacción es un modo de extinguir obligaciones y consiste en un contrato mediante el cual las partes resuelven directamente y por mutuo acuerdo una situación jurídica dudosa.
- Acordar voluntariamente con otra parte algún punto litigioso para compartir la diferencia de la disputa, consentir a fin de terminar con una diferencia.

### Hipótesis: Transacción y trámite
A partir de las primeras definiciones entre trámite y transacción se puede plantear la siguiente hipótesis:

- La transacción podemos asociarla al término de trámite. La transacción necesita 2 o más partes para poder realizarla y el trámite parte de una persona (sea física o jurídica) la cual tiene interacción con la entidad gubernamental. Ambas implican acuerdos y la obtención de algo a cambio.
- ¿Cómo podemos definir a las transacciones? Solicitudes, turnos, pagos, inscripciones, registros, consultas. En todas ellas hay dos partes, se realiza una operación para obtener algo a cambio, por ejemplo: pago por un servicio de luz, brindo mis datos y los de mi partida para obtener una digitalización. 
- Entonces ¿qué podría ser un trámite? Un trámite podría ser un conjunto de transacciones. Por ejemplo: Si tengo que renovar mi licencia, necesito realizar un pago, sacar un turno, realizar una consulta de deuda, etc.. A partir de estas diferentes transacciones yo realizo el trámite de "Renovación de licencia de conducir". 
- ¿Qué pasa si solo necesito realizar una transacción? Por ejemplo: Necesito solicitar mi partida nacimiento y cómo ya tengo todos los datos, completo el formulario y la solicito. En este caso, podríamos llamar trámite, pero para una mejor organización de la información podemos contemplarlo como una **"Solicitud"**. 
- En conclusión todos los trámites tienen transacciones. Si hay 2 o más transacciones (por ejemplo: "Solicitud + pago" o "Inscripción + turno + Pago") podríamos categorizarlo como un **"Trámite"**. Si hay 1 sola transacción podríamos llamarlo por el nombre de la transacción, por ejemplo: **"Pago"**

@Vpappacena @mathiassassone1 @SantiBugdadi @Fcunsolo

**Comment by Vpappacena** [comment]
Excelente que retomamos el punto de partida de México para ajustar el  glosario ,  sumo algo de Colombia que diferencia un trámite de otro procedimiento : https://www.funcionpublica.gov.co/web/suit/40. 

Roseth, Reyes y Santiso (2018): “Los trámites constituyen
el conjunto de requisitos, pasos o acciones a través de los cuales los individuos o las empresas piden o entregan
información a una entidad pública, con el fin de obtener un derecho –generación de un registro, acceso a un servicio,
obtención de un permiso– o para cumplir con una obligación”.

http://tramites.bcs.gob.mx/que-es-un-tramite-o-servicio/

Y sumo el de Argentina de gobar : https://www.argentina.gob.ar/contenidosdigitales/tramites-y-servicios

**Comment by mathiassassone1** [comment]
Agrego algunas diferencias generales.

**Servicios**
- Cualquier intercambio entre el Estado y las personas con el fin de cumplir con una obligación u obtener un beneficio.

**Tramites**
- Varias transacciones.
- Ley que lo respalde.
- Acceso a un derecho.
- Oferta obligatoria para el ciudadano de parte del Estado.
- Solicitud o entrega de información / datos / documentación, etc.
- Presencial, telefónico o digital.

**Transacción**
- Una transacción
- Ley que lo respalde.
- Acceso a un derecho.
- Oferta obligatoria para el ciudadano de parte del Estado.
- Inscripción, registro, solicitud, turno, pago, calendario, consulta.
- Presencial, telefónico o digital.

**Otros**
- Una transacción.
- No necesita ley que lo respalde.
- Programas, becas, evento, taller, actividad, curso, etc
- Potestas de la entidad (Bajo demanda)
- Beneficios o actividades extras.

**Reply by aloyber** [reply]
@mathiassassone1 

Dejo algunas definiciones de **Servicio** que pueden ayudar:
- Un servicio es todo acto o actividad que se ofrece para satisfacer una necesidad
- Trabajo, especialmente cuando se hace para otra persona.
- Los servicios se definen como un conjunto de actividades orientadas a satisfacer las necesidades de un cliente, comunidad o receptor que pueden o no, haber pagado por él.

Si lo trasladamos a nuestro ámbito el **servicio** puede funcionar como algo que ofrece el gobierno para satisfacer una necesidad o cumplir con una obligación. Por ejemplo: algun evento o festival que se realice al aire libre y en un espacio publico, no necesita ningún tipo de intercambio y es algo que ofrece la Ciudad. Hay casos en los que el servicio si necesite un intercambio y casos en los que no.

**Comment by mathiassassone1** [comment, marked as answer]
### Actividades, talleres y cursos en la ciudad:
- Calendarizado: Eventos y ofertas de actividades que cumplen una determinada fecha, horario y lugar. Pueden o no conllevar una transacción.
- No calendarizado: Oferta de actividad, talleres y/o cursos en un espacio digital o presencial. No dependen de una fecha y horario particular. Pueden o no conllevar una transacción.

### Programas Beneficios y becas:
Es un servicio temporal o continuo que ofrece la gestión actual de gobierno, para cubrir un beneficio social, cultural o económico. Conllevan una transacción.

### Trámite:
Es servicio con carácter obligatorio que impone el gobierno para cubrir un necesidad particular del ciudadano. Conlleva una o mas transacciones.

### Materiales y recursos:
Es información complementaria que puede estar dentro de un servicio general ó un material/recurso obligatorio para realizar un trámite.

### Asistencia teléfonos y ayuda:
Acciones que la persona necesita realizar para contactarse y obtener asistencia o ayuda. Es un canal directo con el servicio y puede ser tanto digital como presencial.
